ndn_python_repo/command/repo_commands.py

```diff
@@ -36,4 +36,58 @@ class RepoCommandResponse(TlvModel):
 
 
 class PrefixesInStorage(TlvModel):
-    prefixes = RepeatedField(NameField())
\ No newline at end of file
+    prefixes = RepeatedField(NameField())
+
+
+
+#Catalog Parameters
+
+class CatalogCommandParameter(TlvModel):
+    """
+    Insertion request App Param format.
+    """
+    name = NameField()
+    data_name = NameField()
+
+
+class CatalogResponseParameter(TlvModel):
+    """
+    Catalog Response App Param format.
+    """
+    status = UintField(8)
+
+
+class CatalogRequestParameter(TlvModel):
+    """
+    The data mapping fetch request from a client.
+    """
+    data_name = NameField()
+
+
+class CatalogInsertParameter(TlvModel):
+    """
+    A single Insert Parameter format containing the data name, the name to map and the expiry time.
+    """
+    data_name = NameField()
+    name = NameField()
+    expire_time_ms = UintField(2)
+
+
+class CatalogDeleteParameter(TlvModel):
+    """
+    A single Delete Parameter format containing the data name and the name mapped to be deleted.
+    """
+    data_name = NameField()
+    name = NameField()
```

Comment:
This TLV structure is questionable. See https://github.com/satyaprakash-1729/ndn-python-catalog/issues/1